sklearn/linear_model/tests/test_base.py

```diff
@@ -206,6 +220,61 @@ def test_linear_regression_sparse_multiple_outcome(random_state=0):
     assert_array_almost_equal(np.vstack((y_pred, y_pred)).T, Y_pred, decimal=3)
 
 
+def test_linear_regression_positive():
+    # Test nonnegative LinearRegression on a simple dataset.
+    # a simple dataset
+    X = [[1], [2]]
+    Y = [1, 2]
+
+    reg = LinearRegression(positive=True)
+    reg.fit(X, Y)
+
+    assert_array_almost_equal(reg.coef_, [1])
+    assert_array_almost_equal(reg.intercept_, [0])
+    assert_array_almost_equal(reg.predict(X), [1, 2])
+
+    # test it also for degenerate input
+    X = [[1]]
+    Y = [0]
+
+    reg = LinearRegression(positive=True)
+    reg.fit(X, Y)
+    assert_allclose(reg.coef_, [0])
+    assert_allclose(reg.intercept_, [0])
+    assert_allclose(reg.predict(X), [0])
+
+
+def test_linear_regression_positive_multiple_outcome(random_state=0):
+    # Test multiple-outcome nonnegative linear regressions
```

Comment:
the term we use is "multi-output"

Reply:
Same problem here as before. "multiple-outcome" is used in all other "multi-output" tests. I will not change it now for consistency (readers should not ask themselves if those three tests are about the same kind of output or not). If there is consensus in changing multiple-outcome -> multi-output , the change should be applied on all the correspondent tests. Thanks for your understanding.

Reply:
as you wish, but this is enforcing consistency on a file that is itself inconsistent with the rest of the code base.

Reply:
I totally understand that. So let's change all the inconsistent entries in a new PR.